tests/app/authentication/test_user_id_generator.py

```diff
@@ -0,0 +1,24 @@
+from app.authentication.user_id_generator import UserIDGenerator
+from app.metadata.metadata_store import MetaDataConstants
+from app import settings
+import unittest
+
+
+class TestUserIDGenerator(unittest.TestCase):
```

Comment:
Consider adding a test that verifies that changing the SALT changes the hash for the same values and for different combinations of each element rather than just this one check